### PR TITLE
Getting Started docs: Images and Fonts

### PR DESCRIPTION
**Pull request by delbaoliveira**
Closes: https://linear.app/vercel/issue/DOC-3697/page-images-fonts-and-assets

### CODE REVIEW

File: docs/01-app/01-getting-started/04-images-and-fonts.mdx [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+TODO: Add diagram for `public` folder
+
+## Optmizing images
+
+The Next.js [`<Image>`](/docs/app/building-your-application/optimizing/images) component extends the HTML `<img>` to provide:
+
+- **Size Optimization:** Automatically serving correctly sized images for each device, using modern image formats like WebP and AVIF.
+- **Visual Stability:** Preventing [layout shift](/learn/seo/web-performance/cls) automatically when images are loading.
+- **Faster Page Loads:** Only loading images when they enter the viewport using native browser lazy loading, with optional blur-up placeholders.
+- **Asset Flexibility:** Resizing images on-demand, even images stored on remote servers.
```
Comment (molebox) [inline docs/01-app/01-getting-started/04-images-and-fonts.mdx:26]: I think we can lowercase these. wdyt?
```suggestion
- **Size optimization:** Automatically serving correctly sized images for each device, using modern image formats like WebP and AVIF.
- **Visual stability:** Preventing [layout shift](/learn/seo/web-performance/cls) automatically when images are loading.
- **Faster page loads:** Only loading images when they enter the viewport using native browser lazy loading, with optional blur-up placeholders.
- **Asset flexibility:** Resizing images on-demand, even images stored on remote servers.
```

File: docs/01-app/01-getting-started/04-images-and-fonts.mdx [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+description: Learn how to optimize images and fonts with Next.js
+related:
+  links:
+    - app/api-reference/components/font
+    - app/api-reference/components/image
+---
+
+Next.js comes with automatic image and font optimizations for better performance and user experience. This page will guide you through how to start using them.
+
+## Handling Static Assets
```
Comment (molebox) [inline docs/01-app/01-getting-started/04-images-and-fonts.mdx:13]: Would you define Static Assets as a Vercel term/product name? If not then we can lowercase
```suggestion
## Handling static assets
```

File: docs/01-app/01-getting-started/04-images-and-fonts.mdx [hunk 3]
```diff
@@ -0,0 +1,364 @@
...
+
+You can store static files, like images and fonts, under a folder called `public` in the root directory. Files inside `public` can then be referenced by your code starting from the base URL (`/`).
+
+TODO: Add diagram for `public` folder
+
+## Optmizing images
+
+The Next.js [`<Image>`](/docs/app/building-your-application/optimizing/images) component extends the HTML `<img>` to provide:
```
Comment (molebox) [inline docs/01-app/01-getting-started/04-images-and-fonts.mdx:21]:
```suggestion
The Next.js [`<Image>`](/docs/app/building-your-application/optimizing/images) component extends the HTML `<img>` element to provide:
```

File: docs/01-app/01-getting-started/04-images-and-fonts.mdx [hunk 4]
```diff
@@ -0,0 +1,364 @@
...
+      // blurDataURL="data:..." automatically provided
+      // placeholder="blur" // Optional blur-up while loading
+    />
+  )
+}
+```
+
+Next.js will automatically determine the intrinsic [`width`](/docs/app/api-reference/components/image#width) and [`height`](/docs/app/api-reference/components/image#height) of your image based on the imported file. These values are used to determine the image ratio and prevent [Cumulative Layout Shift](https://nextjs.org/learn/seo/web-performance/cls) while your image is loading.
+
+### Remote Images
```
Comment (molebox) [inline docs/01-app/01-getting-started/04-images-and-fonts.mdx:90]:
```suggestion
### Remote images
```

File: docs/01-app/01-getting-started/04-images-and-fonts.mdx [hunk 5]
```diff
@@ -0,0 +1,364 @@
...
+        pathname: '/my-bucket/**',
+      },
+    ],
+  },
+}
+```
+
+Learn more about Image Optimization in the [API Reference](/docs/app/api-reference/components/image).
+
+## Optimizing Fonts
```
Comment (molebox) [inline docs/01-app/01-getting-started/04-images-and-fonts.mdx:166]:
```suggestion
## Optimizing fonts
```

File: docs/01-app/01-getting-started/04-images-and-fonts.mdx [hunk 6]
```diff
@@ -0,0 +1,364 @@
...
+    </html>
+  )
+}
+```
+
+```jsx filename="app/layout.js" switcher
+import { Geist } from 'next/font/google'
+
+const geist = Geist({
+  subsets: ['latin'],
```
Comment (molebox) [inline docs/01-app/01-getting-started/04-images-and-fonts.mdx:195]: Are we missing `display` here? Its in the TS version of this codeblock

File: docs/01-app/01-getting-started/04-images-and-fonts.mdx [hunk 7]
```diff
@@ -0,0 +1,364 @@
...
+export default function Layout({ children }) {
+  return (
+    <html className={geist.className}>
+      <body>{children}</body>
+    </html>
+  )
+}
+```
+
+### Local Fonts
```
Comment (molebox) [inline docs/01-app/01-getting-started/04-images-and-fonts.mdx:207]:
```suggestion
### Local fonts
```

File: docs/01-app/01-getting-started/04-images-and-fonts.mdx [hunk 8]
```diff
@@ -0,0 +1,364 @@
...
+    {
+      path: './Roboto-BoldItalic.woff2',
+      weight: '700',
+      style: 'italic',
+    },
+  ],
+})
+```
+
+### Google Fonts
```
Comment (molebox) [inline docs/01-app/01-getting-started/04-images-and-fonts.mdx:278]:
```suggestion
### Google fonts
```

**Review by molebox — DISMISSED** [review]

File: docs/01-app/01-getting-started/04-images-and-fonts.mdx (hunk 1 above)
Comment (delbaoliveira) [reply docs/01-app/01-getting-started/04-images-and-fonts.mdx:26]: 💯

File: docs/01-app/01-getting-started/04-images-and-fonts.mdx (hunk 6 above)
Comment (delbaoliveira) [reply docs/01-app/01-getting-started/04-images-and-fonts.mdx:195]: Good catch, thank you 🙏🏼

File: docs/01-app/01-getting-started/04-images-and-fonts.mdx [hunk 9]
```diff
@@ -0,0 +1,364 @@
...
+    </html>
+  )
+}
+```
+
+```jsx filename="app/layout.js" switcher
+import { Geist } from 'next/font/google'
+
+const geist = Geist({
+  subsets: ['latin'],
```
Comment (delbaoliveira) [inline docs/01-app/01-getting-started/04-images-and-fonts.mdx:195]:
```suggestion
  subsets: ['latin'],
  display: 'swap',
```

File: docs/01-app/01-getting-started/04-images-and-fonts.mdx [hunk 10]
```diff
@@ -0,0 +1,371 @@
...
+
+You can store static files, like images and fonts, under a folder called `public` in the root directory. Files inside `public` can then be referenced by your code starting from the base URL (`/`).
+
+<Image
+  alt="Route segments to path segments"
+  srcLight="/docs/light/public-folder.png"
+  srcDark="/docs/dark/public0-folder.png"
+  width="1600"
+  height="282"
+/>
```
Comment (delbaoliveira) [inline docs/01-app/01-getting-started/04-images-and-fonts.mdx:23]:
```suggestion
<Image
  alt="Folder structure showing app and public folders"
  srcLight="/docs/light/public-folder.png"
  srcDark="/docs/dark/public-folder.png"
  width="1600"
  height="282"
/>
```

File: docs/01-app/01-getting-started/04-images-and-fonts.mdx [hunk 11]
```diff
@@ -0,0 +1,371 @@
...
+    />
+  )
+}
+```
+
+Next.js will automatically determine the intrinsic [`width`](/docs/app/api-reference/components/image#width) and [`height`](/docs/app/api-reference/components/image#height) of your image based on the imported file. These values are used to determine the image ratio and prevent [Cumulative Layout Shift](https://nextjs.org/learn/seo/web-performance/cls) while your image is loading.
+
+### Remote images
+
+To use a remote image, you can provide an URL string for the `src` property.
```
Comment (ismaelrumzan) [inline docs/01-app/01-getting-started/04-images-and-fonts.mdx:98]:
```suggestion
To use a remote image, you can provide a URL string for the `src` property.
```

**Review by ismaelrumzan — APPROVED** [review]
one smol typo

**Pull request merged**